### PR TITLE
nuke

### PR DESCRIPTION
**Pull request by loonghao**
support nuke

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,10 +33,12 @@ menu = ScriptsMenu(title="Scripts",
 menu.add_script(parent=menu,
                 title="Script A",
                 command="print('A')",
+                sourcetype='',
```
Comment (BigRoy) [inline README.md]: Well spotted, this should've been fixed in the README. Though I don't believe an empty string is the fix we're looking for. This should be `sourcetype="python"`

File: python/scriptsmenu/action.py [hunk 2]
```diff
@@ -112,7 +112,7 @@ def run_command(self):
             callback(self)
             return
 
-        exec(self.process_command())
+        exec (self.process_command())
```
Comment (BigRoy) [inline python/scriptsmenu/action.py]: Any particular reason you strayed away from PEP08 and added the space?

**Comment by BigRoy** [comment]
@loonghao Great! :) I made two line comments regarding changes, aside from that this is looking great.

Thanks for this!

----

Will you be using this in production?

File: python/scriptsmenu/action.py (hunk 2 above)
Comment (loonghao) [reply python/scriptsmenu/action.py]: OK, later I can check my settings, maybe is fix conflicts lost something

File: README.md (hunk 1 above)
Comment (loonghao) [reply README.md]: Ok. I get it, Thank you.

**Comment by loonghao** [comment]
@BigRoy Not yet, just for myself.

**Comment by BigRoy** [comment]
> Not yet, just for myself.

Feel free to create any issues if you're looking for specific features.

**Comment by loonghao** [comment]
Ok, cool

**Comment by loonghao** [comment]
@BigRoy  can you help me, merger this PR?

**Comment by BigRoy** [comment]
Sure can, will have a look first thing tomorrow.

**Comment by BigRoy** [comment]
Some minor notes still:

- With the readme fix, can you also patch the top example there by adding "python" as the argument for sourcetype.
- Could you version up to 1.2.0 by increasing the minor version in `version.py`?

**Comment by loonghao** [comment]
@BigRoy  already fix

**Pull request merged**